lib/cc/analyzer/container.rb

```diff
@@ -75,13 +75,8 @@ def run(options = [])
         )
       ensure
         t_timeout.kill if t_timeout
```

Comment:
Ah, good point, but, that being the case, I'm not sure the existing code is correct either: `t_out.kill if t_out` will also raise an exception if `t_out` never got defined. We'd want to either use `defined?(t_tout)`, or switch to instance vars that have the nil-if-undefined behavior.